### PR TITLE
Fix Flow v68 errors

### PR DESCRIPTION
**Pull request by hermanventer**
Release note: Add source annotations to satisfy latest version of Flow

Flow now complains about missing properties in more situations.

### CODE REVIEW

File: src/intrinsics/node/utils.js [hunk 1]
```diff
@@ -47,10 +47,10 @@ export function createDeepIntrinsic(realm: Realm, value: mixed, intrinsicName: s
         throw new FatalError("Arrays are not supported yet.");
       }
       let prototype = Object.getPrototypeOf(value);
-      if (prototype !== Object.prototype) {
+      if (prototype !== (Object: any).prototype) {
```
Comment (hermanventer) [inline src/intrinsics/node/utils.js]: This seems like a bug in Flow.

File: src/values/Value.js [hunk 2]
```diff
@@ -47,7 +47,7 @@ export default class Value {
   }
 
   static isTypeCompatibleWith(type: typeof Value, Constructor: typeof Value): boolean {
-    return type.prototype instanceof Constructor || type.prototype === Constructor.prototype;
+    return (type: any).prototype instanceof Constructor || (type: any).prototype === Constructor.prototype;
```
Comment (hermanventer) [inline src/values/Value.js]: Seems like a bug in Flow.

File: src/intrinsics/node/utils.js [hunk 3]
```diff
@@ -47,10 +47,10 @@ export function createDeepIntrinsic(realm: Realm, value: mixed, intrinsicName: s
...
         throw new FatalError(
           "Only simple objects are supported for now. Got: " +
-            ((typeof prototype.constructor === "function" && prototype.constructor.name) ||
+            ((typeof (prototype: any).constructor === "function" && prototype.constructor.name) ||
```
Comment (hermanventer) [inline src/intrinsics/node/utils.js]: Seems like a bug in Flow.

File: src/intrinsics/node/utils.js (hunk 1 above)
Comment (NTillmann) [reply src/intrinsics/node/utils.js]: Cc @avikchaudhuri .

File: src/react/hoisting.js [hunk 4]
```diff
@@ -115,6 +115,7 @@ function canHoistAbstract(realm: Realm, abstract: AbstractValue, residualHeapVis
     for (let scope of scopes) {
       const currentAdditionalFunction = residualHeapVisitor.commonScope;
       invariant(currentAdditionalFunction instanceof FunctionValue);
+      // $FlowFixMe: there is no such property
       if (scope === currentAdditionalFunction.parent) {
```
Comment (NTillmann) [inline src/react/hoisting.js]: So this is likely a bug? Maybe leave behind a TODO?

Comment (NTillmann) [reply src/react/hoisting.js]: I removed the `.parent` property from FunctionValues a while ago as it wasn't well defined. @trueadm, what's the intention behind the `.parent` check? Doesn't seem to affect anything.

**Pull request closed without merging**